### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gustavoguimaraes**
Updated dependencies to reflect newish Solidity compilers and most recent DX smart contracts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,24 +10,24 @@
...
-    "ganache-cli": "^6.1.8",
-    "openzeppelin-solidity": "^1.12.0",
+    "ganache-cli": "^6.4.1",
     "prettier": "^1.16.4",
-    "truffle-contract": "^3.0.6",
-    "truffle-flattener": "^1.2.6",
+    "solc": "0.5.2",
+    "truffle-contract": "^3.0.7",
+    "truffle-flattener": "^1.3.0",
     "truffle-hdwallet-provider": "0.0.5"
```
Comment (elektronaut0815) [inline package.json]: "truffle-hdwallet-provider": "1.0.5"

Comment (gustavoguimaraes) [reply package.json]: Thanks will update

Comment (gustavoguimaraes) [reply package.json]: Updated

**Comment by gustavoguimaraes** [comment]
Merging this so we have the latest stuff in

**Pull request merged**